### ISSUE TITLE
In App Profile Screen

### ISSUE THREAD

**Issue by kistitan** [fetlife/android#188]
Opened to follow up a discussion in a separate topic.

**Comment by kistitan** [comment]
@BarryFruitman 
I added the API call to our Intent Service. It is on master branch.

You can access it with 
FetLifeApiIntentService.startApiCall(context,FetLifeApiIntentService.ACTION_APICALL_MEMBER,memeberId);
It will saved into the Member database table.
And you will be notified about the call finish via EventBus.

Let me know if you needed more help to start playing around with.

Good luck :)

(Note: there are moderate code refactor is going in on branch v1.8 that is going to be merged into master soon.)

**Comment by BarryFruitman** [comment]
Got it. I'm going to start working on this today.

**Comment by kistitan** [comment]
Nice :)

**Comment by BarryFruitman** [comment]
Krisztian did you get my email? The data returned by the Member API very limited and doesn't contain enough information to populate a profile page.

I'd really like to work on the API itself. Is that possible?

**Comment by reneklacan** [comment]
@BarryFruitman I usually work on API backend, please include me in the thread (see my email on GH profile or my first name at fetlife.com) and I'm sure we will be able to help you somehow :)

**Comment by BarryFruitman** [comment]
Done. Thank you!

**Comment by kistitan** [comment]
@BarryFruitman
Do you mean the API or the Member object?
Did you check the actual json? 

I remember that I did not add all the fields to the Member object only those I needed.
If the actual field is in the response json you don't need to modify the API, only to add one more field (with getter/setter) to the Pojo

**Comment by reneklacan** [comment]
@BarryFruitman @kistitan is right, it's probably returned from the API (including about), just check the JSON, here's the example of what it returns to make it easier for you

```json
{
  "websites": [],
  "url": "https://fetlife.com/users/5317298",
  "sexual_orientation": "Not Applicable",
  "role": "Not Applicable",
  "notification_token": "21dd75a6-49fc-4354-8cdd-f267af884b44",
  "nickname": "MagentaRabbit",
  "meta_line": "39 ",
  "looking_for": [],
  "is_supporter": false,
  "id": "14d31dd8c",
  "gender": {
    "name": "Not Applicable",
    "abbreviation": ""
  },
  "friend_count": 4,
  "country": "Argentina",
  "content_type": "member",
  "city": null,
  "avatar": {
    "variants": {
      "small": "https://pic1.fetlife.com/5317/5317298/00052fc4-9d43-af12-8a39-936ce2fe5eb4_60.jpg?token=1477227742_ac185995edbb6865858bc6ece22925e9ea9500362e2cacc348e8dae5b4190f1b",
      "mini": "https://pic2.fetlife.com/5317/5317298/00052fc4-9d43-af12-8a39-936ce2fe5eb4_35.jpg?token=1477227742_c0753c7a0a276a27d6161e7f49d6bd70db5f06890be696bcef0832d7ae29aa5d",
      "medium": "https://pic0.fetlife.com/5317/5317298/00052fc4-9d43-af12-8a39-936ce2fe5eb4_110.jpg?token=1477227742_f9c254f0da4c7fcffd0d8b618de19aef866d1b2ccda8054694472d27af33234a"
    },
    "id": "f3a0b4b3"
  },
  "age": 39,
  "administrative_area": "Chaco",
  "about": "@greenrabbit fads"
}
```

**Comment by kistitan** [comment]
Released in version 2.6.0
